### DISCUSSION TITLE
About loss

### DISCUSSION THREAD

**Discussion by guoqincode** [PixArt-alpha/PixArt-alpha#105] [General]
In contrast to the DiT implementation itself:
https://github.com/facebookresearch/DiT/blob/main/diffusion/gaussian_diffusion.py#L779
            terms["mse"] = mean_flat((target - model_output) ** 2)
            if "vb" in terms:
                terms["loss"] = terms["mse"] + terms["vb"]
            else:
                terms["loss"] = terms["mse"]
                
 Why did you increase the loss by twice as much?
 https://github.com/PixArt-alpha/PixArt-alpha/blob/master/diffusion/model/gaussian_diffusion.py#L814
 
                 terms["mse"] = mean_flat(loss)
            terms["loss"] = terms["mse"] + terms["vb"] if "vb" in terms else terms["mse"]
            if "mae" in terms:
                terms["loss"] = terms["loss"] + terms["mae"]

**Comment by lawrence-cj** [comment]
I didn't get u. Do u think `mse` and `mae` are the same loss, or what?
